README.md

```diff
@@ -1,16 +1,28 @@
 
 ```
 
+# make sure you enabled CORS addon on Minishift
```

Comment:
Could we add this as a step to the makefile rather than in the README for the user to complete?

Reply:
@laurafitzgerald It might be a good idea, but on the other hand we also don't do any `minishift start` in the makefile, which is also cluster startup.

I think we can create a separate make task for minishift though.

Let's address that separately if that's ok